images/linux/scripts/tests/Run-Tests.ps1

```diff
@@ -0,0 +1,23 @@
+param (
+    [Parameter(Mandatory=$true)]
+    $WorkingDirectory,
+    [Parameter(Mandatory=$true)]
+    $OutputFile
+)
+
+$ErrorActionPreference = "Stop"
+
+Import-Module Pester
+Set-Location -Path $WorkingDirectory
+
+$configuration = [PesterConfiguration] @{
+    Output = @{ Verbosity = "Detailed" }
+    TestResult = @{
+        Enabled = $true
+        OutputPath = $OutputFile;
+        OutputFormat = "NUnitXML"
+    }
+}

```

Comment:
I think we should use Windows version of this code because we should fail build if tests are not passed: https://github.com/actions/virtual-environments/blob/main/images/win/scripts/ImageHelpers/TestsHelpers.ps1#L67